swagger-diff.gemspec

```diff
@@ -25,15 +25,16 @@ Gem::Specification.new do |spec|
   spec.executables   = spec.files.grep(%r{^exe/}) { |f| File.basename(f) }
   spec.require_paths = ['lib']
 
-  spec.required_ruby_version = '~> 2.0'
+  spec.required_ruby_version = '>= 2.7'
 
-  spec.add_dependency 'json-schema', '~> 2.6'
+  spec.add_dependency 'json-schema', '~> 3.0'
```

Comment:
ruby 3 support theoretically beings at this version https://github.com/voxpupuli/json-schema/blob/master/CHANGELOG.md